x-pack/test/api_integration/apis/infra/metrics_explorer.ts

```diff
@@ -48,20 +47,20 @@ export default function({ getService }: FtrProviderContext) {
           .set('kbn-xsrf', 'xxx')
           .send(postBody)
           .expect(200);
-        const body: MetricsExplorerResponse = response.body;
+        const body = response.body;
```

Comment:
A quick way to improve instead of removing the typing would be to use the runtime type, which already exists:

```suggestion
        const body = decodeOrThrow(metricsExplorerResponseRT)(response.body);
```

That could save a lot of type-related assertions below.